spec/support/json_helpers.rb

```diff
@@ -0,0 +1,5 @@
+module JsonHelpers
```

Comment:
Instead of this, use `response.parsed_body`. It automagically decodes the json.

Reply:
I did try to use `response.parsed_body` but it ended up parsing the response as a string, even with `as: :json` (`get :bans, params: { id: user.id }, as: :json`). I just created this helper instead 🤷‍♂️

Reply:
It works for the other API requests.